src/character.cpp

```diff
@@ -2046,15 +2046,24 @@ float Character::get_hit_base() const
     return get_dex() / 4.0f;
 }
 
+// @todo: Better place for it?
```

Comment:
output.cpp with the rest of the color functions.

Reply:
That's not my comment, I simply moved this function from end to front, as you can see at the end.

Reply:
No haven't seen that. Could you still move it there or remove the comment.

Reply:
Yeah, I certainly can do that as soon as I get some time to do some fixing in this little project.